### PR TITLE
Fendering non-owner permalink page

### LINKED ISSUE #4131: Add 'do it' button to page
This button will direct people to the campaign pitch page

@sbsmith86 I'm not sure about button styles and stuff, but you would link to 
`drupal_get_path_alias('node/' . $node->nid)`

### PR DESCRIPTION
**Pull request by sbsmith86**
- Refactor permalink template to use a `card` pattern. 
- Updated template to work for the non-owner version of the permalink page. Addresses #4067 
- Added cta to template. Fixes #4131 

@DoSomething/front-end also, should this card pattern be in Neue? Halp.

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss [hunk 1]
```diff
@@ -9,44 +9,41 @@
...
+      @include span(6 of 12);
+      float: none;
+      display: table-cell;
+      vertical-align: top;
+      padding: $base-spacing;
+    }
 
-        &.-border {
-          border-top: 2px solid $light-gray;
+    &.-right {
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss]: Would this make more sense  as a `.card__column + .card__column`?

**Comment by weerd** [comment]
Think overall looks good. Only thing I'm not keen on is the `.-right` modifier class. It implies layout location, but not sure if it should. For example, on mobile, is it on the "right" side? I'm assuming it likely stacks, so the name can be confusing. I would go with something that doesn't imply it's positioning, like giving each column a modifier like, `.-alpha` & `.-beta`, or some other better name.

Minor nit-picking :) Not crucial.

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss (hunk 1 above)
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss]: :+1: to what @DFurnes suggests :)

**Comment by DFurnes** [comment]
Looks nice! :+1: Let's also move `.card` pattern into a `scss/patterns/card.scss` directory with other patterns (not necessary for merge).

**Comment by sbsmith86** [comment]
cool cool. Yea I will make that `-right` change!

**Comment by weerd** [comment]
:dancer:

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss [hunk 2]
```diff
@@ -9,44 +9,42 @@
...
+      @include span(6 of 12);
+      float: none;
+      display: table-cell;
+      vertical-align: top;
+      padding: $base-spacing;
+    }
 
-        &.-border {
-          border-top: 2px solid $light-gray;
+    + .card__column {
```
Comment (sbsmith86) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss:35]: @DFurnes @weerd How does this look?

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss:35]: Do you need the `&` in there?

```
& + .card__column { .. }
```

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss:35]: nevermind. I'm dumb. haha

Comment (sbsmith86) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss:35]: @weerd no, tried it both ways, didn't seem necessary. Also, it's similar to something like this. https://github.com/DoSomething/dosomething/blob/v0.4.27/lib/themes/dosomething/paraneue_dosomething/scss/patterns/_promotion.scss#L89-L94

**Pull request merged**